### DISCUSSION TITLE
wsl --list --online doesn't show Alpine in the list

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/WSL#6190] [Q&A] [answered]
![Screenshot of wsl --list --online not listing Alpine](https://user-images.githubusercontent.com/67918878/97991937-0fe26900-1e08-11eb-827c-4cd18e04f1d5.png)

`wsl --list --online` does not show Alpine in list. I would like to know why.

**Comment by Biswa96** [comment, marked as answer]
Because it's not provided officially by Alpine developers/maintainers.

**Comment by nbetcher** [comment]
For those **not** immediately concerned with the **source of the WSL version of Alpine Linux that is available from the Microsoft Store**, you may manually install Alpine Linux for WSL by following these directions if you are unable to use the Microsoft Store GUI itself: https://github.com/microsoft/WSL/discussions/6276#discussioncomment-1084583

Otherwise, if you want to acquire the same, **unofficial** version of Alpine Linux for WSL by means of the Microsoft Store GUI, click here: https://www.microsoft.com/en-us/p/alpine-wsl/9p804crf0395
